contracts/tokenomics/maker/src/contract.rs

```diff
@@ -540,10 +542,16 @@ fn distribute(
         let amount =
             amount.multiply_ratio(Uint128::from(cfg.governance_percent), Uint128::new(100));
         if amount.u128() > 0 {
-            let to_governance_asset = token_asset(cfg.astro_token_contract.clone(), amount);
-            result.push(SubMsg::new(
-                to_governance_asset.into_msg(&deps.querier, governance_contract)?,
-            ))
+            let send_msg = CosmosMsg::Wasm(WasmMsg::Execute {
+                contract_addr: cfg.astro_token_contract.to_string(),
+                msg: to_binary(&Cw20ExecuteMsg::Send {
+                    contract: governance_contract.to_string(),
+                    msg: to_binary(&escrow_fee_distributor::Cw20HookMsg::Burn {})?,
+                    amount,
+                })?,
+                funds: vec![],
+            });
+            result.push(SubMsg::new(send_msg))
```

Comment:
The Maker does not have a message reply handler. Let's use regular messages instead submessages.